low-level/framework/base/sspl_constants.py

```diff
@@ -277,6 +278,7 @@ class RaidAlertMsgs(Enum):
 class AlertTypes(Enum):
     GET = "get"
     FAULT = "fault"
+    FAULT_RESOLVED = "fault resolved"
```

Comment:
`fault_resolved` is what we have used everywhere, it should be same eveywhere

Reply:
Done.